mmdet/models/utils/shape_convert.py

```diff
@@ -0,0 +1,28 @@
+def nlc_to_nchw(x, hw_shape):
```

Comment:
Is it necessary to wrap it to a function?

Reply:
We can move these function to `mmdet/models/utils/transformer.py `